frontend/src/libs/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -0,0 +1,62 @@
+import { Link } from '#libs/components/components.js';
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+import { useCallback, useLocation, useState } from '#libs/hooks/hooks.js';
+import { type Routes } from '#libs/types/types.js';
+
+import styles from './styles.module.scss';
+
+const Dropdown: React.FC<Routes> = ({ routes }) => {
```

Comment:
Component name should be the same as file name
```suggestion
const DropdownMenu: React.FC<Routes> = ({ routes }) => {
```